zerver/views/webhooks/mention.py

```diff
@@ -0,0 +1,36 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from typing import Dict, Any, Iterable, Optional
+
+@api_key_only_webhook_view('Mention')
+@has_request_variables
+def api_mention_webhook(request, user_profile, client,
+                        payload=REQ(argument_type='body'),
+                        stream=REQ(default='Mention'),
+                        topic=REQ(default='News')):
```

Comment:
We usually use lower-case streams and topics for the defaults in webhook integrations.